.svglintrc.js

```diff
@@ -129,6 +132,36 @@ module.exports = {
               }
             }
           },
+          function(reporter, $, ast) {
+            reporter.name = "icon-precision";
+
+            const iconPath = $.find("path").attr("d");
+            const { segments } = parsePath(iconPath);
+            const segmentParts = segments.flat().filter((num) => (typeof num === 'number'));
+
+            const countDecimals = (num) => {
+              let text = num.toString();
+              if (text.indexOf('e-') > -1) {
+                let [base, trail] = text.split('e-');
+                let elen = parseInt(trail, 10);
+                let idx = base.indexOf('.');
+                return idx == -1 ? 0 + elen : (base.length - idx - 1) + elen;
+              }
+              let index = text.indexOf('.');
+              return index == -1 ? 0 : (text.length - index - 1);
```

Comment:
Looks good, but doesn't work for exponential positives numbers right? I possible that someone could add a exponential positive in a path? I don't know. I have been experimenting with this function for supporting exponential positive numbers:

```javascript
if (num && num % 1) {
    let [base, op, trail] = num.toExponential().split(/e([+-])/);
    let elen = parseInt(trail, 10);
    let idx = base.indexOf('.');
    return idx == -1 ? elen : base.length - idx - 1 + (op === '+' ? -elen : elen);
  }
  return 0;
```

Reply:
LGTM! Surely more flexible this way!